docker/wiremock/stubs/__files/correctPatientStructuredRecordResponse.json

```diff
@@ -1 +1 @@
38916FFCDE3F"}],"status":"completed","medicationReference":{"reference":"Medication/AEA9506C-7524-45CD-A88B-E4FE8F65EBB8"},"effectivePeriod":{"start":"2010-01-13","end":"2018-10-27"},"dateAsserted":"2010-01-13T11:37:34.423+00:00","subject":{"reference":"Patient/88F14BF6-CADE-47D6-90E2-B10519BF956F"},"taken":"unk","dosage":[{"text":"n/a"}]}},{"resource":{"resourceType":"MedicationStatement","id":"EB4A7657-6AD0-4418-B187-3BF64EC2FD80-MS","meta":{"profile":["https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-MedicationStatement-1"]},"extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1","valueCodeableConcept":{"coding":[{"system":"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1","code":"prescribed-at-gp-practice","display":"Prescribed at GP practice"}]}}],"identifier":[{"system":"https://EMISWeb/A82038","value":"8AA68359-DA7D-101B-F7F4-BCA90A2BFF5F"}],"basedOn":[{"reference":"MedicationRequest/EB4A7657-6AD0-4418-B187-3BF64EC2FD80"}],"status":"completed","medicationReference":{"reference":"Medication/7A2E347E-CC28-47D9-8717-117A38C51B7B"},"effectivePeriod":{"start":"2010-01-13","end":"2018-10-27"},"dateAsserted":"2010-01-13T11:37:34.423+00:00","subject":{"reference":"Patient/88F14BF6-CADE-47D6-90E2-B10519BF956F"},"taken":"unk","dosage":[{"text":"One To Be Taken Each Day"}]}},{"resource":{"resourceType":"MedicationStatement","id":"47784A83-5675-4EF2-9947-0D7D125F1656-FD-1-MS","meta":{"profile":["https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-MedicationStatement-1"]},"extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1","valueCodeableConcept":{"coding":[{"system":"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1","code":"prescribed-at-gp-practice","display":"Prescribed at GP practice"}]}},{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatementLastIssueDate-1","valueDateTime":"2010-10-20T00:00:00+01:00"}],"identifier":[{"system":"https://EMISWeb/A82038","value":"57E4E63C-DEFE-B7BB-2652-4BB20FA31362"}],"basedOn":[{"reference":"MedicationRequest/47784A83-5675-4EF2-9947-0D7D125F1656-FD-1"}],"status":"completed","medicationReference":{"reference":"Medication/9CA00C4D-E545-4A18-BA84-C74D74DD7E95"},"effectivePeriod":{"start":"2010-01-13","end":"2010-11-17"},"dateAsserted":"2010-01-13T11:37:34.423+00:00","subject":{"reference":"Patient/88F14BF6-CADE-47D6-90E2-B10519BF956F"},"taken":"unk","dosage":[{"text":"One Tablet Daily For 21 Days; Subsequent Courses Repeated After 7-Day Tablet Free Interval"}]}},{"resource":{"resourceType":"MedicationStatement","id":"581EC356-84CA-450B-9D0A-DADEAAD743D3-FD-1-MS","meta":{"profile":["https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-MedicationStatement-1"]},"extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1","valueCodeableConcept":{"c
```

Comment:
We intend for our wiremock to mirror the patients that are in the GP connect demonstrator.

Instead of replacing the data that is already there, could you add this as an additional patient? I assume it is a test patient from one of the example datasets. Could you make the NHS number for the wiremock patient the same as the NHS number for the example dataset?

Reply:
Would it be enough to add "Encounter" and "List" resources?

Reply:
I would recommend creating an ASR with just the encounter list and encounter resources for the unit test. We should use complete patient records for the wiremock.